metaworld/policies/sawyer_push_wall_v2_policy.py

```diff
@@ -0,0 +1,61 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, move
+
+
+class SawyerPushWallV2Policy(Policy):
+
```

Comment:
`from metaworld.policies.policy import assert_fully_parsed` and use it as a decorator here

Reply:
Added!